generator/src/test/java/com/scottlogic/deg/generator/cucumber/longerThan.feature

```diff
@@ -0,0 +1,671 @@
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+
+  Scenario: Running an 'longerThan' request alongside a after constraint should be successful
+    Given foo is longer than 1
+    And foo is after 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.001" |
+      |  2019-01-01T00:00:00.001   |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.001" |
+      |  2019-01-01T00:00:00.001   |
+
+
+  Scenario: Running an 'longerThan' request alongside a after constraint should be successful
+    Given foo is longer than 1
+    And foo is anything but after 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "a"    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      |  2019-01-01T00:00:00.000   |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      |  2019-01-01T00:00:00.000  |
+
+
+  Scenario: Running an 'longerThan' request alongside a afterOrAt test1  constraint should be successful
+    Given foo is longer than 1
+    And foo is after or at 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "a"    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      | "2018-09-01T00:00:00.001" |
+      | 2019-01-01T00:00:00.000   |
+      | 2019-01-01T00:00:00.001   |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      | "2018-09-01T00:00:00.001" |
+      | 2019-01-01T00:00:00.000   |
+      | 2019-01-01T00:00:00.001   |
+
+
+  Scenario: Running an 'longerThan' request alongside a  afterOrAt test2  constraint should be successful
+    Given foo is longer than 1
+    And foo is anything but after or at 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "a"    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2018-02-01T00:00:00.001" |
+      | "2019-01-01T00:00:00.000" |
+      | 2018-01-01T00:00:00.001   |
+      | 2019-01-01T00:00:00.000   |
+    Then the following data should be generated:
+      | foo    |
+      | null   |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2018-02-01T00:00:00.001" |
+      | "2019-01-01T00:00:00.000" |
+      |  2018-01-01T00:00:00.001  |
+
+  Scenario: Running a 'longerThan' request using a number (decimal number) to specify a the length of a generated string should fail with an error message
```

Comment:
This is a duplicate of the scenario at line 38?

Reply:
Yea it was a duplicate test. I have removed it